pkgs/applications/qubes/qubes-core-vchan-xen/default.nix

```
@@ -25,6 +25,7 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
+    broken = true;
```

```suggestion
    broken = true;
```
The meta parameters have [order](https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes).
`broken` is last. Reason for that is to avoid git conflicts, it's always last.

---

Reordered the other Xen meta attributes and fixed the broken tags.